Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -1768,6 +1768,22 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
       .AddParameter("expression",
                     _("Angle of tolerance, in degrees (0: minimum tolerance)"))
       .AddCodeOnlyParameter("conditionInverted", "")
+      .SetHidden()
+      .MarkAsAdvanced();
+
+  extension
+      .AddCondition("IsTurnedTowardObject",
+                    _("An object is turned toward another"),
+                    _("Check if an object is turned toward another"),
+                    _("_PARAM0_ is turned towards _PARAM1_ ± _PARAM2_°"),
```

Comment:
```suggestion
                    _("_PARAM0_ is turned toward _PARAM1_ ± _PARAM2_°"),
```
To be consistent with the instruction name and description